modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -106,6 +107,8 @@ protected Object resolvePropertyToExample(String mediaType, Property property, S
             return 1.23f;
         } else if (property instanceof IntegerProperty) {
             return 123;
+        } else if (property.getClass().equals(BaseIntegerProperty.class)) {
```

Comment:
I see. Languages may have different needs when it comes to default values. `NodeJSServerCodegen` does not override the `toDefaultValue` method.

Do you think it is enough to have this extra case in `ExampleGenerator`? I think `toDefaultValue` in `DefaultCodegen` should become abstract so each language must implement their own default values.